CHANGELOG.md

```diff
@@ -1,3 +1,239 @@
+<!--
+# YY.0M (YYYY-0M-0D)
+
+Breaking changes:
+
+Features:
+
+Commands:
+
+Usability improvements:
+
+Fixes:
+
+Themes:
+
+New languages:
+
+Updated languages and queries:
+
+Packaging:
+-->
+
+# 25.01 (2025-01-01)
+
+`git log` says there are 164 contributors since 24.07
+
+checkpoint <https://github.com/helix-editor/helix/compare/a5a7cff311e909ecab41f840742a5309634aeec3...master>
+
+<!-- Breaking changes: -->
+
+Features:
+
+* Big refactor for `Picker`s (#9647, #11209, #11216, #11211, #11343, #11406)
+    * Use a table layout and allow filtering by column
+    * Reimplement `global_search` to allow changing the query dynamically
+* Add an alternative "inline" display for LSP diagnostics (#6417, #11815)
+* Support defining keybindings as macros (#4709)
+* Continue line comments in `o`/`O` and on `<ret>` in insert mode (#10996, #12213, #12215)
+* Allow configuring and switching clipboard providers at runtime (#10839, b855cd0, 467fad5, 191b0f0)
+* Add support for path completion (#2608)
+* Support bindings with the Super (Cwd/Win/Meta) modifier (#6592)
+* Support rendering and jumping between tabstops in snippet completions (#9801)
+* Allow theming directory completions (#12205, #12295)
+
+Commands:
+
+* Add commands to move within snake_case or camelCase words (#8147)
+* Add `search_selection_detect_word_boundaries` (#12126)
+    * This command takes the `*` key in normal and select mode, replacing `search_selection` which was moved to `A-*`.
```

Comment:
wouldn't this be a breaking change? albeit small

Reply:
We only call changes that break existing configs a breaking change. Otherwise every improvement endsup a breaking change